src/fastga_he/models/propulsion/components/loads/sm_pmsm/components/perf_air_gap_windage_losses.py

```diff
@@ -0,0 +1,161 @@
+# This file is part of FAST-OAD_CS23-HE : A framework for rapid Overall Aircraft Design of Hybrid
+# Electric Aircraft.
+# Copyright (C) 2025 ISAE-SUPAERO
+
+import numpy as np
+import openmdao.api as om
+
+from ..constants import DEFAULT_DENSITY
+
+
+class PerformancesAirGapWindageLosses(om.ExplicitComponent):
+    """
+    Computation of the air gap windage losses result from frictions of the air gap between the rotor
+    and the stator bore. This is obtained from equation (II.70) in :cite:`touhami:2020`.
+    """
+
+    def initialize(self):
+        self.options.declare(
+            name="motor_id", default=None, desc="Identifier of the motor", allow_none=False
+        )
+        self.options.declare(
+            "number_of_points", default=1, desc="number of equilibrium to be treated"
+        )
+
+    def setup(self):
+        motor_id = self.options["motor_id"]
+        number_of_points = self.options["number_of_points"]
+
+        self.add_input("rpm", units="min**-1", val=np.nan, shape=number_of_points)
+        self.add_input(
+            "density",
+            shape=number_of_points,
+            val=DEFAULT_DENSITY,
```

Comment:
Since it is an input that is supposed to be coming from the mission, I don't mind not having a default value. It would be an indicator of th good connection